actor/builtin/paymentbroker/paymentbroker.go

```diff
@@ -334,6 +341,49 @@ func (pb *Actor) Extend(vmctx exec.VMContext, chid *types.ChannelID, eol *types.
 	return 0, nil
 }
 
+// Cancel can be used to end a storage deal early. It lowers the EOL of the
```

Comment:
Could/Should the comment be made more general? I believe the paymentbroker will handle more than payments just for storage deals. Maybe something like: 

>"Cancel can be used to end an **off chain payment** early. It lowers the EOL of the payment channel to 1 blocktime from now and allows a **caller** to reclaim their payments. In the time before the channel is closed, a **target** can potentially dispute a closer."